cfme/infrastructure/datastore.py

```diff
@@ -112,8 +112,8 @@ def get_detail(self, *ident):
     def _on_detail_page(self):
         """ Returns ``True`` if on the datastore detail page, ``False`` if not."""
         return sel.is_displayed(
-            '//div[@class="dhtmlxInfoBarLabel-2"][contains(., "{}") and contains(., "{}")]'.format(
-                self.name, "Summary")
+            '//div[@id="main-content"][contains(., "{}") and contains(., "{}")]'.format(
+                "Datastore", self.name)
```

Comment:
Please, use `cfme.web_ui.summary_title` instead, if it does not get matched, add the additional locator in the SUMMARY_TITLE_LOCATORS.